sdk/eventhubs/azure-messaging-eventhubs-track1-perf/src/main/java/com/azure/messaging/eventhubs/perf/EventProcessorOptions.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs.perf;
+
+import com.beust.jcommander.Parameter;
+
+/**
+ * Options for Event Processor tests.
+ *
+ * @see EventProcessorTest
+ */
+public class EventProcessorOptions extends EventHubsReceiveOptions {
+    @Parameter(names = {"-scs", "--storageConnectionString"}, description = "Connection string for Storage account.",
```

Comment:
Do we allow a test that use sample checkpoint store without a storage container?

Reply:
The problem is that if we want something comparable, we need to do the same implementation for Track 1... I had tried it for a few hours but it was too confusing to continue.